nle/env/base.py

```diff
@@ -569,16 +569,18 @@ def render(self, mode="human"):
                     )
             except ValueError:  # inv_strs/letters not used.
                 pass
-            colors_index = self._observation_keys.index("colors")
-            chars = self.last_observation[chars_index]
-            colors = self.last_observation[colors_index]
+
+            chars = self.last_observation[self._observation_keys.index("chars")]
+            colors = self.last_observation[self._observation_keys.index("colors")]
             print(nethack.tty_render(chars, colors))
             return
-        elif mode == "ansi":
-            chars = self.last_observation[chars_index]
+
+        if mode in ("ansi", "string"):  # Misnomer: This is the least ANSI of them all.
+            chars = self.last_observation[self._observation_keys.index("chars")]
+            # TODO: Why return a string here but print in the other branches?
```

Comment:
Returning a string is useful for things like WANDB, loggers, etc i think

Reply:
But it kinda breaks the API (and pylint doesn't like it).

But okay.